decorator.py

```diff
@@ -0,0 +1,33 @@
+"""
+This task contains a few parts:
+Write a custom function which returns your name or username.
+Write a decorator which allows access for your user and prints 
+�Permission denied� for other users.
+
+Tips:
+Create get_user_info function which returns your name or username.
+Create decorator which raises Exception if get_user_info returns not 
+your name or username
+Create check_perms function decorated by decorator. This function
+returns Allowed access if user validation is successful.
+"""
+
+
+
+
+def decorator(my_name):
+    def wrapper(f):
+        if not my_name:
```

Comment:
Well, how this decorator works?

Reply:
if nothing is passed, then the permission is forbidden

Reply:
and if the data transmitted, then the permission is allowed

Reply:
Well, if I'll pass `['sergiy']` this decorator also allows access for `pavlo`, what is not expected.

Reply:
done